opentracing-spring-cloud-starter-jaeger/pom.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2017-2018 The OpenTracing Authors
+
+    Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+    in compliance with the License. You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software distributed under the License
+    is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+    or implied. See the License for the specific language governing permissions and limitations under
+    the License.
+
+-->
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.opentracing.contrib</groupId>
+    <artifactId>opentracing-spring-cloud-parent</artifactId>
+    <version>0.1.8-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>opentracing-spring-cloud-starter-jaeger</artifactId>
+
+  <properties>
+    <main.basedir>${project.basedir}/..</main.basedir>
+
+    <jaeger.version>0.26.0</jaeger.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>opentracing-spring-cloud</artifactId>
```

Comment:
I am wondering whether it should rather depend on the starter module.

Reply:
yes please. It seems that the pattern is to have one core module and starter. The core depends on cores/(and some starters) and starter depends on starters. 

At least this is in sleuth.

Reply:
I am not sure what are the benefits of having core and -stater. Perhaps some dependency manipulation?

Reply:
This is also mentioned in the Spring Boot [docs](https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-developing-auto-configuration.html)

Reply:
Updated dependency